modules/local/plot_entity_binding_ratios.nf

```diff
@@ -0,0 +1,35 @@
+process PLOT_ENTITY_BINDING_RATIOS {
+
+    conda (params.enable_conda ? "bioconda::bioconductor-alphabeta:1.8.0--r41hdfd78af_0" : null)
```

Comment:
conda never with build (here: `--r41hdfd78af_0`) because depending on the environment, different builds are used and the way it is declared here will fail on same infrastructures.